content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+MITRE for example proposes an end-to-end framework to preserve the integrity of the software supply chain. OpenSSF develops the SLSA framework that groups several security best practices for open-source projects.
+
+Moreover, the in-toto Attestation Framework defines a fixed, lightweight Statement that communicates some information about the execution of the software supply chain, such as if the source was reviewed, or if it went through a SLSA conformant build process.
+
+Supply chain security is important to be constantly increased through safeguards placed as soon as possible in the supply chain ("shift left") and needs to be maintained in a continuous manner reducing the operational burden (DevSecOps) because technologies evolve so attacks do.
+ 
+Software artifact signatures are one of the safeguards with a reasonable amount of effort nowadays, thanks also to important projects like Sigstore's [cosign](https://github.com/sigstore/cosign) for artifacts distributed as container images. So this was one of the goals decided for the Falco Software Supply Chain Working Group.
+
+## Artifacts distributed as OCI
+
+Since January 2023 we started distributing Falco plugins and rules as [OCI artifacts](https://github.com/oras-project/artifacts-spec), in addition to other ways. If you are not familiar with the concept, think about OCI artifacts as a content-addressable unit of data that can be tagged, stored and indexed in container registries alongside your container images but can hold any type of file. You can find the source and the data for Falco's artifacts in the [plugins](https://github.com/falcosecurity/plugins) and [rules](https://github.com/falcosecurity/rules) repositories respectively. We are using GitHub Container Registry to publish them so you can find them in the "Package" section of each repository.
+
+This brings several advantages, such as:
+* We can create multiple tags, both fixed and floating, for each artifact. Keeping track of the `latest` version is easy as well as the latest version for each major and minor release.
+* Distribution is handled with a standard protocol implemented by many different container registries, which means that creating [your own rules registry](https://falco.org/blog/gitops-your-falco-rules/), either public or private, is easy and you can use your existing container distribution infrastructure (or any ready-to-use registry service as offered by all major cloud providers) without installing and maintaining any custom server.
+* There are standard and developing mechanisms and specifications to store artifact metadata, like [link between artifacts](https://opencontainers.org/posts/blog/2023-07-07-summary-of-upcoming-changes-in-oci-image-and-distribution-specs-v-1-1/#2-new-manifest-field-for-establishing-relationships) for signature or SBOM references, so we do not need to reinvent the wheel.
```

Comment:
With both SBOM here and SLSA above - either link to something that that explains what they are or use their full name and put the acronym in parenthesis if you're going to use it again.